protobuf/grpc_service.proto

```diff
@@ -673,8 +673,8 @@ message ModelInferRequest
   //@@     match what is expected by the tensor's shape and data
   //@@     type. The raw data must be the flattened, one-dimensional,
   //@@     row-major order of the tensor elements without any stride
-  //@@     or padding between the elements. Note that the FP16 data
-  //@@     type must be represented as raw content as there is no
+  //@@     or padding between the elements. Note that the FP16 and BF16 data
+  //@@     types must be represented as raw content as there is no
```

Comment:
@deadeyegoodwin do these changes require a parity change with https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#inference-1 ?

Reply:
Yes, you will want to submit a PR to the kserve project to update the docs to include BF16.